modules/tinymce/src/plugins/table/main/ts/actions/TableActions.ts

```diff
@@ -68,14 +68,17 @@ export const TableActions = (editor: Editor, lazyWire: () => ResizeWire, selecti
   // Optional.none gives the default cloneFormats.
   const cloneFormats = getCloneElements(editor);
 
+  const colMutationOp = isResizeTableColumnResizing(editor) ? Fun.identity : CellMutations.halve;
```

Comment:
```suggestion
  const colMutationOp = isResizeTableColumnResizing(editor) ? Fun.noop : CellMutations.halve;
```
This would be more consistent with `CellMutations.halve` as it's return type is `void`.